src/hooks/useFavorites.ts

```diff
@@ -0,0 +1,18 @@
+import { axiosInstance, Favorites_URLS } from "../services/urls";
+import useFetch from "./useFetch";
+
+export default function useFavorites() {
+  const getFavorites = async () => {
+    const response = await axiosInstance.get(Favorites_URLS.Get_Fav);
```

Comment:
A good practice when using the api response is to provide the correct type for it. This will make working with the response easier and less error prone.